app/helpers/arclight_helper.rb

```diff
@@ -110,11 +110,11 @@ def grouped?
   end
 
   def search_with_group
-    search_state.params_for_search.merge('group' => 'true')
+    search_state.params_for_search.merge('group' => 'true').reject { |k| k == 'page' }
```

Comment:
Can we use [`#except`](https://apidock.com/rails/Hash/except) here?
```suggestion
    search_state.params_for_search.merge('group' => 'true').except('page')
```